content/articles/introduction-to-subnetting/index.md

```diff
@@ -0,0 +1,170 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /introduction-to-subnetting/
+title: Introduction to Subnetting
+description: Subnetting is a partition process of splitting a single-page network into two or more strands. This process enhances the efficiency of the computer network. This article will give good detail on how subnetting works and the terminologies behind computer networking.
+author: anyebe-blessing-ene
+date: 2021-05-20T00:00:00-10:30
+topics: []
+excerpt_separator: <!--more-->
+images:
+  -url: /engineering-education/introduction-to-subnetting/hero.jpg
+  alt: Subnetting image example
+---
+In this article, we will get to understand important terms of the Network.
+<!--more-->
+### Table of Contents
+- Internet Protocol (IP) Address
+- IPV4 Diagram
+- IP Classes
+- Public vs Private IP
+- Classless Inter-Domain Routing (CIDR)
+- Broadcast ID 
+
+### Prerequisites
+One simply requires basic network knowledge of the internet. 
+
+Let’s dive right in.
+
+**Subnetting** is the process of partitioning a network into two or more networks based on utilization. It is a logical process of splitting a single-page network into more strands. Subnetting came into the internet in 1981. 
+
+A protocol is a set of standards and rules that define the language that devices can use to communicate. There are several existing protocols used in networking such as TCP, UDP, IP and, ICMP. In this article, we will only discuss the IP.
+
+### Internet Protocol address (IP Address)
+
+An IP is a distinctive identifier. It is the foundational protocol that allows the internet to work. It is situated in the internet layer of the TCP/IP model. There are  32- bits numbers.
+For example; 216.58.216.164 (Google’s IPV4 IP).
+
+#### IPV4 Structure (Internet Protocol Version 4)
+
+IPV4 is the first version of IP for the internet. The [Defense Advanced Research Projects Agency](https://www.darpa.mil) developed IPV4 in 1981. They deployed for use in the ARPANET in 1983. 
+
+An IPV4 is a 32-bits logical number displayed in decimal notation and divided into 8bits each (1 octet). 
+
+A particular address consists of; 
+
+1. Host identity 
+
+2. Network identity.
+
+Below is a diagram of IPV4
+
+[Diagram](/introduction-to-subnetting-diagram-of-IPV4.jpeg)
```

Comment:
For the article to look as presentable as possible, is it possible to get a clearer image?